symbionts/pressbooks-latex/pb-latex.php

```diff
@@ -137,7 +137,12 @@ function &latex( $latex, $background = false, $color = false, $size = 0 ) {
 		if ( ! $color )
 				$color = empty( $this->options['fg'] ) ? '000000' : $this->options['fg'];
 
-		require_once( dirname( __FILE__ ) . "/automattic-latex-{$this->methods[$this->options['method']]}.php" );
+		if ( file_exists( dirname( __FILE__ ) . "/automattic-latex-{$this->methods[$this->options['method']]}.php" ) ) {
+			require_once( dirname( __FILE__ ) . "/automattic-latex-{$this->methods[$this->options['method']]}.php" );
+		} else {
+			apply_filters( 'pb_require_latex', $this->methods[$this->options['method']] );
```

Comment:
Needs documentation, as per: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Additionally I'm wondering if it might be better to call this `pb_require_latex_renderer` for clarity. Otherwise looks good! 👍

Reply:
Actually, scratch that. I realize the hook is already documented elsewhere. On an unrelated note it might be a good idea to do the same thing that we do [here](https://github.com/pressbooks/pressbooks/blob/dev/symbionts/pressbooks-latex/pb-latex-admin.php#L197-L199), e.g.

```
if ( 'wpcom' == method) {
  require_once( dirname( __FILE__ ) . "/automattic-latex-wpcom.php" );
} else {
```